Doc/Orbiter User Manual/LAUNCHPAD.tex

```diff
@@ -172,61 +137,336 @@ \subsection{Visual effects tab}
 
 \begin{figure}[H]
 	\centering
 	\subfigure{\includegraphics[width=0.95\textwidth]{planet_tile_res.jpg}}
 	\caption{Kennedy Space Center at resolution level 12 (left) and 19 (right)}
 \end{figure}
-
 \end{itemize}
 
 \noindent
 \\
 \textbf{General effects}
-
 \begin{itemize}
 \item \textbf{Vessel shadows:} Enable shadows cast by spacecraft on planet surfaces.
-\item \textbf{Object shadows:} Enable dynamic shadows of ground-based objects such as buildings.
-\item \textbf{Specular reflection from objects:} Render reflective surfaces like solar panels, window panes and metallic surfaces. May degrade performance.
 \item \textbf{Reentry flames:} Render a glowing plasma hull during reentry.
+\item \textbf{Object shadows:} Enable dynamic shadows of ground-based objects such as buildings.
 \item \textbf{Particle streams:} Render ionised exhaust gases and vapour trails with particle effects.
+\item \textbf{Specular object reflection:} Render reflective surfaces like solar panels, window panes and metallic surfaces. May degrade performance.
 \item \textbf{Local light sources:} Enable localised light sources, e.g. from engines, landing lights, floodlights etc. This option can have a significant influence on frame rates.
 \item \textbf{Ambient light level:} Defines the brightness of the unlit side of planets and moons. Ambient level 0 is the most realistic, but makes it difficult to spot objects in the dark. Level 255 is uniform lighting (no darkness).
 \end{itemize}
 
+\subsubsection{Physics settings}
+\begin{figure}[H]
+	\centering
+	\subfigure{\includegraphics[width=0.25\textwidth]{launchpad_physics.png}}
+\end{figure}
+
+\textbf{Perturbations}
+\begin{itemize}
+\item \textbf{Nonspherical gravity sources:} This option activates a more complex gravity force calculation which can take into account perturbations in the gravitational potential due to non spherical celestial body shapes, thus allowing more accurate orbit predictions. Note that this option can make orbital calculations more difficult, and may reduce the reliability of instruments that don't take this effect into account. For a planet to make use of the perturbation code, its configuration file must supply shape parameters via the \textit{JCoeff} entry. For background and technical implementation details please refer to "Nonspherical gravitational field perturbations" in Orbiter Technical Reference.
```

Comment:
"must supply shape parameters via the JCoeff entry"^, or provide a Spherical Harmonic model with the _GravModelPath_ and _GravCoeffCutoff_ entries

Reply:
Fixed, thanks!